planner/view.go

```diff
@@ -26,7 +26,9 @@ type viewNode struct {
 }
 
 func (p *Planner) View(query *mapper.Select, desc client.CollectionDescription) (*viewNode, error) {
-	m, err := mapper.ToSelect(p.ctx, p.db, desc.BaseQuery)
+	baseQuery := (desc.Sources[0].(*client.QuerySource)).Query
```

Comment:
todo: Same comment as the `0` index one.

Reply:
Same answer as the other question